pipeline/splicing/calcVarPriors.py

```diff
@@ -245,7 +259,10 @@ def getExonBoundaries(variant):
     Uses function implemented in calcMaxEntScanMeanStd to get data for variant's transcript
     '''
     varTranscript = variant["Reference_Sequence"]
-    transcriptData = fetch_gene_coordinates(str(varTranscript))
+    if varTranscript == BRCA1_RefSeq:
```

Comment:
the three identical blocks can all be a function call, e.g. something like:
```
def getTranscriptData(referenceSequence):
    if varTranscript == BRCA1_RefSeq: 
            return brca1TranscriptData
    elif varTranscript == BRCA2_RefSeq:
            return brca2TranscriptData
```

then just call it when you need it:
```
transcriptData = getTranscriptData(varTranscript)